packages/calcite-components/src/components/functional/Heading.tsx

```diff
@@ -17,7 +17,7 @@ export const Heading: FunctionalComponent<HeadingProps> = (props, children): VNo
   delete props.level;
 
   return (
-    <HeadingTag class={props.class} key={props.key} level={props.level}>
```

Comment:
`level` prop doesn't seem to exist in the DOM?
was it used for styling or testing? if not, we should remove it

---

original discussion in https://github.com/Esri/calcite-design-system/pull/10352#discussion_r1767097375